lucene/core/src/java/org/apache/lucene/util/automaton/createLevAutomata.py

```diff
@@ -22,7 +22,7 @@
 import os
 import sys
 # sys.path.insert(0, 'moman/finenight/python')
-sys.path.insert(0, '../../../../../../../../build/core/moman/finenight/python')
+sys.path.insert(0, '../../../../../../../../../build/moman/finenight/python')
```

Comment:
I think this has been answered before, but please remind me, does this break `ant regenerate` meaning that both cannot co-exist?

Reply:
Yeah. I'm pretty sure it does break the ant regenerate. Since the whole regenerate process is apparently run extremely rarely (like every couple of years or so from what I can tell), I think we'll be on gradle exclusively the next time this is run.

Reply:
If so then we should add a fail to ant and just say "use gradle?". These python scripts could take an argument - the path to moman passed from gradle script. Then it'd be elegant and clear without those ugly relative paths.

Reply:
Dawid:

Yeah, I wondered about that. But please don't make me run the whole regenerate task in ant ;).

I'll change this to take an argument and get it working in Gradle, and leave a comment in the python code about having to change things a bit if running from Ant.